### PR TITLE
docs: Update get started java sdk steps

### PR DESCRIPTION
**Pull request by FengShang**
Update Java SDK install guide.

image
image

### CODE REVIEW

File: src/pages/getStarted/components/constants.ts [hunk 1]
```diff
@@ -257,7 +259,7 @@ export const getJSCode = (options: IOption) => {
       code: 
 `import { FeatureProbe, FPUser } from "featureprobe-client-sdk-js";
 
-const userId = /* unique user id in your business logic */;
+const userId = /* User id, for percentage rollout */;
 const user = new FPUser(userId);
 ${userWithCode}
 const fp = new FeatureProbe({
```
Comment (frankzhaopeng) [inline src/pages/getStarted/components/constants.ts:265]: Missed a change in line 277

Comment (frankzhaopeng) [reply src/pages/getStarted/components/constants.ts:265]: @FengShang

File: src/locales/en-US.json [hunk 2]
```diff
@@ -305,10 +305,13 @@
   "connect.fourth.failed": "You have no applications connected using this SDK Key. ",
   "connect.fourth.success": "Successfully connected！",
   "connect.fourth.failed.button": "Retry Connection",
-  
-  "getstarted.java.first.step": "Step 1: Create a new project（optional）",
-  "getstarted.java.second.step": "Step 2: Add the SDK to your project",
-  "getstarted.java.third.step": "Step 3: Add the following code to your Application",
+
+  "getstarted.java.first.step": "Step 1： Install the Java SDK to your project",
```
Comment (frankzhaopeng) [inline src/locales/en-US.json]: Use English Punctuation 「:」 not「 ：」

File: src/locales/en-US.json [hunk 3]
```diff
@@ -305,10 +305,13 @@
...
+  "getstarted.java.first.step.name.one": "Use your exist project, or create a new one as follow:",
+  "getstarted.java.first.step.name.two": "Add FeatureProbe Java SDK to dependency",
+  "getstarted.java.second.step": "Step 2：Create a single FeatureProbe Client instance for project",
```
Comment (frankzhaopeng) [inline src/locales/en-US.json]: Use English Punctuation 「:」 from line 312 to 314

**Comment by hezean** [comment]
IMHO there isn't a method `FeatureProbe#close()` in the java sdk?

**Comment by FengShang** [comment]
> IMHO there isn't a method `FeatureProbe#close()` in the java sdk?

We are working on it, will be added soon.

**Pull request merged**